Makefile

```diff
@@ -0,0 +1,51 @@
+VENV := $(PWD)/env
+PYTHON := python
+PIP := $(PYTHON) -m pip
+
+.PHONY: help venv3 clean
+
+help:	# The following lines will print the available commands when entering just 'make'
+ifeq ($(UNAME), Linux)
+	@grep -P '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | \
+		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+else
+	@awk -F ':.*###' '$$0 ~ FS {printf "%15s%s\n", $$1 ":", $$2}' \
+		$(MAKEFILE_LIST) | grep -v '@awk' | sort
+endif
+
+venv3: ### Creates a virtual environment for this project
+	test -d $(VENV) || $(PYTHON) -m venv $(VENV) && source $(VENV)/bin/activate
+	$(PIP) install --upgrade pip
+	$(PIP) install -e ".[dev,test]"
```

Comment:
honestly, i would kill the venv stuff, personally speaking i'm using conda and i feel more control using conda rather then venv.
for a public, it can feel like we are forcing this.

although, it doesn't hurt anyone 😂  so

Reply:
up to what the others decide

Reply:
@aquemy , @andre-lx what you think?

Reply:
The problem I see is that `venv` is still way more `conda` which might make it easier for most people to install the SDK using the makefile.

Reply:
Yes, I also agree with @aquemy , at least for now